### PR TITLE
"dev" build with Python 3 support

### LINKED ISSUE #23: Python 3
Why is pynio only available on Python 2?

### PR DESCRIPTION
**Pull request by khallock**
Closes #23 and #36

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
 
 package:
   name: pynio
   version: {{ version }}
 
 source:
-  url: https://github.com/NCAR/pynio/archive/{{ version }}.tar.gz
-  sha256: e2bae6469541662835a7d13aa0185b73b13df56c2de6ce6f8e74cb17ba9a6143
+  git_url: https://github.com/NCAR/pynio.git
+  git_rev: develop
```
Comment (ocefpaf) [inline recipe/meta.yaml]: We should use the commit hash here so we can reproduce the build.

Comment (khallock) [reply recipe/meta.yaml]: Good call, I just updated it to use the commit hash.

Sorry for not using the other pull request, I didn't notice you had made one until after I had already created this one.

Comment (ocefpaf) [reply recipe/meta.yaml]: > Sorry for not using the other pull request, I didn't notice you had made one until after I had already created this one.

No problem. I only sent that as an example. Thanks for moving this forward!

**Comment by ocefpaf** [comment]
Let's merge this so we can test it with xarray. Thanks @khallock!

**Pull request merged**